app/src/main/java/com/projectfkklp/saristorepos/repositories/UserRepository.java

```diff
@@ -30,7 +30,7 @@ public static  void getSignedInUser(SignInMethod signInMethod, OnUserRetrieve ca
 
     public static void getCurrentUser(OnUserRetrieve callback){
         getCollectionReference()
-            .document(AuthenticationRepository.getCurrentUserUid())
+            .document(AuthenticationRepository.getCurrentAuthenticationUid())
             .get()
             .addOnCompleteListener(task -> {
                 DocumentSnapshot document = task.getResult();
```

Comment:
rename this function into getUserByCurrentAuthentication

for more concise information that we are getting the use to the current authenticated user